.github/dependabot.yml

```diff
@@ -0,0 +1,23 @@
+version: 2
+updates:
+  - package-ecosystem: "gitsubmodule"
+    directory: "/"
+    schedule:
+      interval: daily
+    open-pull-requests-limit: 99
+    reviewers:
+      - "WikiRik"
```

Comment:
you can also add me and Fabrizio here I think? either that or AEGEE/everybody group

Reply:
I didn't want to do AEGEE/everybody so not to spam too much, but I can change it if you want